exercises/nucleotide-count/HINTS.md

```diff
@@ -0,0 +1,3 @@
+##HINTS
```

Comment:
Please add a space between the hashes and the text, and also use non-caps text (so that would become "Hints")